app/ui/legacy/src/main/java/com/fsck/k9/ui/fab/HideFabOnScrollBehavior.kt

```diff
@@ -22,10 +23,14 @@ class HideFabOnScrollBehavior(context: Context, attributes: AttributeSet) :
         super.onAttachedToLayoutParams(lp)
     }
 
+    // FIXME restricted API
```

Comment:
Trading a lint warning for a FIXME comment doesn't feel like a win. I've created PR #7692 to address this warning.

Reply:
I added the FIXME as I wasn't aware of the reasoning behind the use of this restricted API. Good that it's available now. I updated #7692 to patch this pull-request.

Reply:
The FIXME comment and the `@SuppressLint` annotation are still present.

I think it would have been easier to drop the "Fix restricted API lint error by suppressing it" commit from this PR instead of trying to merge the two PRs.